app/livechat/client/views/app/livechatDepartmentForm.js

```diff
@@ -171,26 +178,31 @@ Template.livechatDepartmentForm.events({
 	'click .remove-agent'(e, instance) {
 		e.preventDefault();
 
-		let selectedAgents = instance.selectedAgents.get();
-		selectedAgents = _.reject(selectedAgents, (agent) => agent._id === this._id);
-		instance.selectedAgents.set(selectedAgents);
+		let departmentAgents = instance.departmentAgents.get();
```

Comment:
if you add multiple agents, then you click to remove one of them, it is removing everyone you just added.

from what I've seen, the agents you just added don't have an `_id`, so `this._id` on the line below is empty.

I'd change to use `agentId` (as I saw every record has) and use `departmentAgents.filter( ... )` instead of `_.reject`